src/main_app/jobs_workers/public_jobs_workers/copy_svg_langs/steps/inject_one_file.py

```
@@ -104,3 +104,10 @@ def inject_step_one_file(
         )
 
     return injects_result
+
+
+__all__ = [
+    "InjectResult",
+    "start_injects",
+    "inject_step_one_file",
+]
```

Ruff (0.15.17)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)